cinema/models.py

```diff
@@ -84,23 +84,35 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
+    @staticmethod
+    def validate_ticket(
+            row: int,
+            seat: int,
+            cinema_hall: CinemaHall,
+            errors_to_raise
```

Comment:
The parameter `errors_to_raise` in the `validate_ticket` method is not documented. It's important to document all parameters to ensure clarity on what is expected, especially since this parameter is used to raise exceptions.